README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Requirements
 
-  * Ruby 1.9.3 or above.
+  * Ruby 2.0.0 or above.
```

Comment:
Looks like 1.9.3 support was dropped 4 years ago: https://www.ruby-lang.org/en/news/2015/02/23/support-for-ruby-1-9-3-has-ended/